wemake_python_styleguide/violations/consistency.py

```diff
@@ -2197,3 +2199,30 @@ class FloatZeroViolation(TokenizeViolation):
 
     code = 358
     error_template = 'Found a float zero (0.0)'
+
+
+@final
+class AssignToSliceViolation(ASTViolation):
+    """
+    Forbid assignment to a subscript slice.
+
+    Reasoning:
+        We do this for consistency.
```

Comment:
Consistency is not the main reason here. Hard to spot bugs and list that changes its size implicitly and strangely are our reasons.